cernopendata_client/cli.py

```diff
@@ -362,3 +364,27 @@ def verify_files(server, recid):
         msg_type="success",
         msg="Success!",
     )
+
+
+@cernopendata_client.command()
+@click.option("--path", type=click.STRING, help="EOS PUBLIC Path")
+def list_directory(path):
+    """List file in a directory.
```

Comment:
List contents of a EOSPUBLIC Open Data directory. 

Returns the list of files and subdirectories of a given EOSPUBLIC directory.